cdcsdk-testing/pom.xml

```diff
@@ -177,6 +177,28 @@
         <artifactId>google-cloud-pubsub</artifactId>
         <scope>test</scope>
     </dependency>
+    <dependency>
+        <groupId>com.amazonaws</groupId>
+        <artifactId>aws-java-sdk</artifactId>
+        <version>1.11.1015</version>
+        <scope>test</scope>
+    </dependency>
+    <dependency>
+        <groupId>com.amazonaws</groupId>
+        <artifactId>amazon-kinesis-client</artifactId>
+        <version>1.0.0</version>
+        <scope>test</scope>
+    </dependency>
+    <dependency>
+        <groupId>com.fasterxml.jackson.core</groupId>
+        <artifactId>jackson-annotations</artifactId>
+        <version>2.13.2</version>
```

Comment:
You'll have to change it to `2.13.4` to keep it consistent with connector. Just made that change there a few mins ago

Reply:
Sure, should I make a change to 2.13.4 [here](https://github.com/yugabyte/cdcsdk-server/blob/main/pom.xml#L78) too